pkg/controller/controlplane/valuesprovider.go

```diff
@@ -389,7 +389,7 @@ func (vp *valuesProvider) GetControlPlaneShootChartValues(
 	if err != nil {
 		return nil, err
 	}
-	return vp.getControlPlaneShootChartValues(ctx, cpConfig, cp, cloudProfileConfig, cluster, secretsReader, checksums)
+	return vp.getControlPlaneShootChartValues(ctx, cpConfig, cp, cloudProfileConfig, cluster, secretsReader)
```

Comment:
`make verify` locally fails for me with:
```
> Check
Executing golangci-lint
pkg/controller/controlplane/valuesprovider.go:378:2: unused-parameter: parameter 'checksums' seems to be unused, consider removing or renaming it as _ (revive)
	checksums map[string]string,
	^
1 issues:
* revive: 1
```

I don't know why it is not failing on GittHub actions? 😮 
Ref https://github.com/gardener/gardener-extension-provider-openstack/actions/runs/16569056551/job/46856186550?pr=1101

@ccwienk @kon-angelo any idea?